dev/com.ibm.ws.jaxrs.2.x.monitor/bnd.bnd

```diff
@@ -0,0 +1,53 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: JAX-RS 2.X Monitor
+Bundle-SymbolicName: com.ibm.ws.jaxrs.2.x.monitor
+Bundle-Description: IBM JAX-RS 2.x monitor; version=${bVersion}
+
+
+WS-TraceGroup: JAXRS
+
+Import-Package: \
+    com.ibm.websphere.servlet.container, \
+    javax.servlet; version="[2.6,3)", \
+    !*.internal.*,*
+
+Export-Package: \
+    com.ibm.websphere.jaxrs.monitor; type="ibm-api", \
+    com.ibm.ws.jaxrs.monitor
+    
+Private-Package: \
+	com.ibm.websphere.jaxrs
```

Comment:
You should be able to delete these lines.  I don't see anything in the `com.ibm.websphere.jaxrs` package other than the `com.ibm.websphere.jaxrs.monitor` package which is exported.

Reply:
Done